sdks/java/container/boot.go

```diff
@@ -122,6 +122,7 @@ func main() {
 		filepath.Join(jarsDir, "beam-sdks-java-harness.jar"),
 		filepath.Join(jarsDir, "beam-sdks-java-io-kafka.jar"),
 		filepath.Join(jarsDir, "kafka-clients.jar"),
+		filepath.Join(jarsDir, "beam-sdks-java-io-snowflake.jar"),
```

Comment:
You should not need this. You just have to make sure that dependencies correctly get determined (and staged) during cross language expansion.

Reply:
I removed it, thanks.